sn_interface/src/messaging/system/agreement.rs

```diff
@@ -66,6 +65,11 @@ impl DkgSessionId {
         hash
     }
 
+    /// A small checksum of the session id's hash used for logging as it is very short
+    pub fn sum(&self) -> u16 {
+        self.hash().iter().map(|i| *i as u16).sum::<u16>()
+    }
+
```

Comment:
Give this a more descriptive name, something like `dbg_short_hash`. `sum` makes it seem like something general / important.

Also, you'll get fewer collisions if you just take the prefix of the hash, i.e. if you want 16 bits, take the first two bytes of the hash.
```suggestion
    /// A small checksum of the session id's hash used for logging as it is very short
    pub fn dbg_short_hash(&self) -> u16 {
        u16::from_le_bytes(self.hash()[..2])
    }

```

Reply:
or maybe `sh()`